### PR TITLE
Prevent spinning play wheel in iOS Safari

### PR DESCRIPTION
**Pull request by robwalch**
Throttling state change updates and toggling the dragging `.jw-flag-dragging` within state updates insures that css flags are set at once and not reset synchronously.

Fixes JW7-3233

### CODE REVIEW

File: src/js/view/view.js [hunk 1]
```diff
@@ -976,19 +953,27 @@ define([
 
         function _stateHandler(model, state) {
             _currentState = state;
-
+            // Throttle all state change UI updates except for play to prevent iOS 10 animation bug
```
Comment (johnBartos) [inline src/js/view/view.js]: Why don't we use _.throttle()?

http://underscorejs.org/#throttle

**Comment by sidpatel13** [comment]
test this please

File: src/js/view/view.js (hunk 1 above)
Comment (robwalch) [inline src/js/view/view.js]: Because we also want to cancel all timeouts when the view is destroyed.

**Comment by johnBartos** [comment]
test this please

**Comment by johnBartos** [comment]
Going to a new playlist item via the playlist interface doesn't show the buffering icon anymore

**Comment by robwalch** [comment]
@johnBartos maybe if you throttle. Hopefully state changes are smoother now and if playback starts right away the buffering state doesn't show while playback is just starting.

**Pull request merged**